### ISSUE TITLE
Add private API

### ISSUE THREAD

**Issue by opiswahn** [Phyyl/Cryptopian#1]

**Comment by Phyyl** [comment]
Yeah I'll be working on it this weekend, come back sunday I should have it

**Comment by jjziets** [comment]
Which Sunday was that? ;-)

**Comment by Phyyl** [comment]
Unfortunately I switch platforms and I'm no longer using Cryptopia for my trading activites. You're more than welcome to fork/PR.
